### ISSUE TITLE
1.20.1 Blocks updating/placing slow when across chunk borders.

### ISSUE THREAD

**Issue by Razatop** [MCRcortex/nvidium#141]
When using Nvidium 0.2.5 and crossing chunks, blocks, doors and other items can take a few seconds to update to show the correct state or for example for torches to pop up.

https://youtu.be/ydOFBbGlArk

https://paste.gg/p/anonymous/813df1b4d6674a9cb04cf364b7db74fd

**Comment by Briarbot2011** [comment]
yeah i'm having the same problem, i was about to make a new issue about it lol

**Comment by MCRcortex** [comment]
Compatibility issue with bobby (same as #156 )
